micro_mg3_0.F90

```diff
@@ -1038,8 +1055,24 @@ subroutine micro_mg_tend ( &
   real(r8) :: faloutr(nlev)
   real(r8) :: faloutnr(nlev)
   real(r8) :: falouts(nlev)
-  real(r8) :: faloutns(nlev)
 
+  real(r8) :: faloutns(nlev)
+  real(r8) :: faltndc
+  real(r8) :: faltndnc
+  real(r8) :: faltndi
+  real(r8) :: faltndni
+  real(r8) :: faltndqie
+  real(r8) :: faltndqce
+
+  real(r8) :: faltndr
+  real(r8) :: faltndnr
+  real(r8) :: faltnds
+  real(r8) :: faltndns
+
+  real(r8) :: faloutg(nlev)
+  real(r8) :: faloutng(nlev)
```

Comment:
shall we initialize the `faloutns`, `faloutg` and `faloutng`?

Reply:
These do not seem to be used anymore in the code. Removed.